ONSdigital/ras-frontstage
# Use correct dockerfile

# What and why?

When it builds and releases the image on a merge to main, it was using the Dockerfile found in the root, rather than the one found in _infra/docker/Dockerfile.

This PR fixes it to use the right one, and also tweaks the settings slightly to more closely match response-operations-ui.
Theres a chance how we run the app might need a few tweaks as we've not had this app be used with gunicorn in a prod-like environment before.

I'm confident it should work as is, as for the dev environments, it DOES use the correct dockerfile so we know it at least runs.

# How to test?

Deploy to your environment and use it like you normally would.  Run the acceptance tests against it too.  Should work as before.

# Trello
